src/System.Runtime.Intrinsics/ref/System.Runtime.Intrinsics.cs

```diff
@@ -9,12 +9,13 @@
 
 namespace System.Runtime.Intrinsics
 {
+    [StructLayout(LayoutKind.Sequential, Size = 8)]
+    public struct Vector64<T> where T : struct {}
     [StructLayout(LayoutKind.Sequential, Size = 16)]
     public struct Vector128<T> where T : struct {}
     [StructLayout(LayoutKind.Sequential, Size = 32)]
     public struct Vector256<T> where T : struct {}
 }
-
 namespace System.Runtime.Intrinsics.X86
```

Comment:
Might be good to have one file per namespace
I was adding ARM64 intrinsics to a separate file

Reply:
> Might be good to have one file per namespace

I agree. However, it doesn't appear any of the other reference sources do this...

Reply:
I was wondering the same thing. My initial thought was that the others are in a single file because they are generated by a tool. But maybe we consciously made the decision to have a single file for the whole ref assembly.

@stephentoub @weshaggard @ericstj @KrzysztofCwalina - thoughts/opinions?

Reply:
Personally, I find the current ref C# files hard to read. There are no line breaks between namespaces/classes and no breaks between things like properties vs methods/etc.

Even if being in a single file is a conscious decision, it might be worthwhile changing the format slightly to make it more readable.